File: app/src/main/java/it/corelab/airbooks/mFragments/LibraryFragment.java
package it.corelab.airbooks.mFragments;

import android.app.Fragment;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.ActionBar;
import android.support.v7.app.AppCompatActivity;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import java.util.zip.Inflater;

import it.corelab.airbooks.R;

/**
 * Created by Roberto_Vecchio on 02/02/18.
 */

public class LibraryFragment extends android.support.v4.app.Fragment {


    //=======================
    //  costructor required
    //=======================

    public LibraryFragment() {
    }

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedIstance){
        View rootView = inflater.inflate(R.layout.library_fragment, container, false);
        return rootView;
    }
}

File: app/src/main/java/it/corelab/studios/navigation/fragment/LibraryFragment.kt
package it.corelab.studios.navigation.fragment

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import it.corelab.studios.airbooks.R

class LibraryFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? =
            View.inflate(activity, R.layout.library_fragment, null)

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        //
    }
}